### PR TITLE
A project to display my portfolio page. I modify the mobile menu bar to toggle and display other options when you click on it using java-script

### PR DESCRIPTION
**Pull request by chukwuebukaVictor**
Modify linters file
Add empty script.js file
Modify linters.yml file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,80 +20,61 @@
...
+      </span>
+      <span class="xIcon">
+        x
+      </span>
+    </button>
+      <ul class="mobile-menu">
+        <li><a class="mobileLink" href="#">Portfolio</a></li>
+        <li><a class="mobileLink" href="#">About</a></li>
+        <li><a class="mobileLink" href="#">Contact</a></li>
+      </ul>
```
Comment (Davidosky007) [inline index.html]: * Kindly fix the mobile menu design as your is different from the given design template 
* [ORIGINAL]
image
* [YOURS]
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @lfmnovaes @mike2611 @chukwuebukaVictor 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* Kindly Write a meaningful title for your pull request and add a short summary of changes introduced in the pull request this helps the reviewer know the changes that were introduced in the pull request. [Check here on how to make a proper pull request](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 2]
```diff
@@ -20,80 +20,61 @@
...
+    <header class="menu-bar">
+      <a class="img-logo" href="#">my logo</a>
+      <ul class="desktop-menu">
+        <li><a href="#">portfolio</a></li>
+        <li><a href="#">about</a></li>
+        <li><a href="#">contact</a></li>
+      </ul>
+      <button class="ham" type="button">
+        <span class="menuIcon">
+          <img src="../img/menuIcon.svg" alt="menu-icon">
```
Comment (hemantso) [inline index.html]: - [ ] The hamburger menu button is not displaying correctly. kindly try to fix it.

image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text
 ## Required Changes ♻️
_Check the comments under the review._


Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — CHANGES_REQUESTED** [review]
Hello Team👋
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You submitted a CR request for other project (details popup window) on the dashboard and PR is at a different milestones. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

------

**Review by oxenprogrammer — APPROVED** [review]
Hi @chukwuebukaVictor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Optional Changes**
- [ ] The title of the Pull Request should be precise and concise. You may then explain what you have done in the body of the Pull Request.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**